libs/blocks/brick/brick.js

```diff
@@ -90,6 +90,12 @@ function decorateBricks(el) {
 }
 
 export default async function init(el) {
+  const regex = /rounded-corners/g;
+  if ([...el.classList].some((c) => regex.test(c))) {
```

Comment:
Is a regex actually needed? Would `el.className.includes('rounded-corners')` not achieve the same result? From a performance perspective, it would also be preferable.
<img width="980" alt="Screenshot 2024-01-10 at 11 19 56" src="https://github.com/adobecom/milo/assets/11267498/3aa565df-f2f6-4c37-bf92-fbb447833d81">

Reply:
it is working as expected. Made the same change for media block as well. Please review it again